jcommune-model/src/test/java/org/jtalks/jcommune/model/validation/IntegerRangeValidatorTest.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (C) 2011  JTalks.org Team
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.jtalks.jcommune.model.validation;
+
+import org.jtalks.jcommune.model.validation.annotations.IntegerRange;
+import org.jtalks.jcommune.model.validation.validators.IntegerRangeValidator;
+import org.mockito.Mock;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+/**
+ * @author skythet
+ */
+public class IntegerRangeValidatorTest {
+
+    @Mock
+    private IntegerRange integerRange;
+
+    @BeforeMethod
+    public void init() {
+        initMocks(this);
+    }
+
+    @Test
+    public void testValidate() {
+        IntegerRangeValidator validator = new IntegerRangeValidator();
+
+        when(integerRange.min()).thenReturn(1);
+        when(integerRange.max()).thenReturn(15);
```

Comment:
Also, instead of using concrete integers let's use Datagen to generate random values: https://bitbucket.org/qala/datagen/